internal/core/repository/src/main/kotlin/com/recco/internal/core/repository/mapper/StyleMapper.kt

```diff
@@ -0,0 +1,14 @@
+package com.recco.internal.core.repository.mapper
+
+import com.recco.api.model.ReccoFont
+import com.recco.api.model.ReccoPalette
+import com.recco.api.model.ReccoStyle
+import com.recco.internal.core.openapi.model.StyleDTO
+
+internal fun StyleDTO.asEntity() = ReccoStyle(
+    font = ReccoFont.valueOf(androidFont.value.uppercase()),
```

Comment:
not sure if this could lead into an exception for old clients in case the API is updated returning other values, should we wrap it in a try/catch for example?

Reply:
I see, maybe we can set a default font in the mapper instead of propagating the error, but doing this we lost track of the actual error, so I think is fine